docs/5_2_hybrid_co-simulation_api.adoc

```diff
@@ -47,7 +45,7 @@ If `canReturnEarly` is `fmi3False` the FMU will not do the early return, regardl
 Note that only the first discontinuity event at a Newtonian time instant shall be signaled that way.
 But in *Event Mode*, there may be an event iteration at a Newtonian time instant and have super dense time instants.
 
-Based on the information delivered in <<fmi3IntermediateUpdateInfo>>, additional information about the discontinuity at that time instant can be obtained from the <<fmi3EventInfo>> structure after calling  <<fmi3NewDiscreteStates>>  and/or <<fmi3GetClock>>.
+Based on the information provided by in <<fmi3CallbackIntermediateUpdate>>, additional information about the discontinuity at that time instant can be obtained by calling <<fmi3NewDiscreteStates>> and <<fmi3GetClock>>.
 
 The FMU must not call the <<fmi3CallbackIntermediateUpdate>> callback function with a `intermediateUpdateInfo.intermediateUpdateTime` value that is smaller than the `intermediateUpdateInfo.intermediateUpdateTime` value given in a previous call of <<fmi3CallbackIntermediateUpdate>> with `intermediateUpdateInfo.intermediateStepFinished` is `fmi3True`.
```

Comment:
Multiple occurences in this and other lines of `intermediateUpdateInfo.intermediateUpdateTime` and `intermediateUpdateInfo.intermediateStepFinished` and other `intermediateUpdateInfo.* need to be changed by removing the intermediateUpdateInfo. part.